package.json

```diff
@@ -87,13 +87,14 @@
     "chalk": "^2.3.0",
     "chrome-remote-interface": "^0.30.0",
     "coffeescript": "^2.3.1",
-    "commander": "^8.0.0",
+    "commander": "^8.3.0",
```

Comment:
Yes, we can ask for a valid token again. I'm ok with the both options.

Reply:
Yes, we can ask for a valid token again. I'm ok with the both options.

Reply:
At present, token validation logic is simple: get 200 HTTP - validation passed, else - not.

We need to be sure that token validation is what is failed the request. Now, the validation API call can be failed by several reasons:
* dashboard/AWS routing issue (for example, `Handler not found` error)
* unhandled error into the target method
* token validation issue

It requires additional logic from TestCafe Dashboard.

Reply:
@kirovboris

Reply:
We've decided keep it as is. Since we have copy button in the dashboard, there's a little chance that copying goes wrong.
In other word - we print a dashboard error if the code is not 200 and shut down the cli session.